UIforETW/UIforETWDlg.cpp

```diff
@@ -1646,7 +1646,7 @@ void CUIforETWDlg::PreprocessTrace(const std::wstring& traceFilename)
 		child.Run(bShowCommands_, L"xperf.exe" + args);
 		std::wstring output = child.GetOutput();
 		std::map<std::wstring, std::vector<DWORD>> pidsByType;
-		for (auto& line : split(output, '\n'))
+		for (const auto line : split(output, '\n'))

```

Comment:
It took me a while to figure out why it should be a reference - it *looks* a bit like we're assigning a return value to a reference, maybe it'd be better to store the result of split in a variable, and then iterate over it? 

i.e.:

```
auto lines = split(output, '\n');
for (const auto& line : lines)
```